packages/gcf-utils/test/integration/gcf-bootstrapper-integration.ts

```diff
@@ -14,9 +14,32 @@
 
 import {GCFBootstrapper} from '../../src/gcf-utils';
 import {describe, beforeEach, afterEach, it} from 'mocha';
-import {Application} from 'probot';
+import {Application, GitHubAPI} from 'probot';
 import {resolve} from 'path';
 import {config} from 'dotenv';
+import assert from 'assert';
+
+/**
+ * How to run these tests:
```

Comment:
@bcoe @chingor13 @orthros I was able to get these tests to finally run following these instructions - I added them in here in case someone else runs into the same issue, but I don't know if this is the 'official'/'correct' way of doing this. Let me know if this sounds okay!

Reply:
This seems reasonable to me. Many integration tests require setup in a test project (since most won't have direct access to the production project resources).